settings/base.py

```diff
@@ -162,4 +162,7 @@
 
     # Set to True to enable this Xblock in mobile apps.
     'ENABLE_ORA_MOBILE_SUPPORT': False,
+
+    # Set to True to enable this features
```

Comment:
Update this comment to explain what the feature does, bonus points if you link to an epic
```suggestion
    # Set to True to enable copying rubric data from existing ORAs <JIRA-ticket>
```